### ISSUE TITLE
Need copyright owner

### ISSUE THREAD

**Issue by ShonFrazier** [campoy/embedmd#62]
https://github.com/campoy/embedmd/blob/c59ce00e02967db5e93d897330142a3ef6145786/LICENSE#L189

Your README claims the code is owned by Google. The license should contain the name of the copyright owner.

**Comment by campoy** [comment]
That's incorrect, sorry. The files owned by Google have the corresponding header. The only place where {yyyy} {name of copyright owner} have been kept is the boilerplate in the license file.
